ADDING_LEARN_PAGE.md

```diff
@@ -0,0 +1,95 @@
+# Adding a Learn Page
```

Comment:
This should be referenced on CONTRIBUTING.md :)

Reply:
You mean a link or a part of `CONTRIBUTING.md` ?

Reply:
Part of CONTRIBUTING.md (as a reference)